components/net/filemanager_thread.rs

```diff
@@ -7,6 +7,7 @@ use mime_guess::guess_mime_type_opt;
 use net_traits::blob_url_store::{BlobBuf, BlobURLStoreError};
 use net_traits::filemanager_thread::{FileManagerResult, FileManagerThreadMsg, FileOrigin, FilterPattern};
 use net_traits::filemanager_thread::{FileManagerThreadError, ReadFileProgress, RelativePos, SelectedFile};
+#[cfg(any(target_os = "macos", target_os = "linux", target_os = "windows"))]
 use servo_config::opts;
```

Comment:
What about moving the `use` to the function that actually uses it instead?

Reply:
Yep, I can do that. Was not sure if that was the common choice in Servo. We could also move the tinydialog one then